telos-backend/README.md

```diff
@@ -0,0 +1,6 @@
+# Backend Readme
```

Comment:
We probably want to leave this for the currently open PR for the README right? But not a big deal if added in this PR.

Reply:
I will address the open PR